### ISSUE TITLE
The readme on npm is 0.4.0

### ISSUE THREAD

**Issue by TheGrandmother** [benwiley4000/gif-frames#12]
The readme on npm is still for version `0.4.0` :p

Thanks for a dank library otherwise!

**Comment by benwiley4000** [comment]
Yeah... I forgot to update it until after the release. It seemed to me like it would be overkill to do another release just for up-to-date docs on the npm site?

**Comment by benwiley4000** [comment]
I guess I'll close this whenever there's a new release :)

**Comment by TheGrandmother** [comment]
I agree that might be a bit overkill, but the links are to 0.4.0 so lazy copy-pasters like me will run around with an older version :p

**Comment by benwiley4000** [comment]
Fair enough.. I'll try to come up with an excuse to release again soon :)

**Comment by TheGrandmother** [comment]
I'm using this for a project and I have found a bunch of bugs ^-^ Will make issues of them soon, so they might make for good candidates for a new release :)

**Comment by benwiley4000** [comment]
@TheGrandmother please do open issues for any bugs you find! It's much appreciated. :) My worst nightmare is that people find bugs in my code and don't report them lol

**Comment by TheGrandmother** [comment]
Haha I know the feeling ^^ I'll make the issues later this afternoon :)
But I don't judge you. Gif may be one of the most horrible and pointless formats every.
It is ancient and was intended to be used for small 5-10 framed animations when 256 colors was more than enough. It is lossy AF and the file sizes are something of a meme :p

**Comment by benwiley4000** [comment]
Depending on the bug it might be a problem with an underlying dependency (the only "decoding" we do directly here is overlaying progressive gif frames) but if you find something, report it here and we'll sort out the root cause.

Thanks again!
